### DISCUSSION TITLE
moose function

### DISCUSSION THREAD

**Discussion by Oops-Qiao** [idaholab/moose#23415] [Q&A General] [answered]
Hi,

May I ask a general question about the moose.
If now considering a problem between a 1D model 2D model, what is the function for a point.

Thanks very much.

Regards,
Jerry

**Comment by GiudGiud** [comment, marked as answer]
Hello

You can use the positions attribute of the MultiApp to translate the the app's frame of reference if they are not well aligned.

If you are transferring a field/ a variable's value I would just use a MultiAppNearestNodeTransfer

Guillaume

**Reply by Oops-Qiao** [reply]
>

**Reply by GiudGiud** [reply]
You can use positions. This has nothing to do with the mesh
Please just try it

no you cant ignore the relative positions of systems. They need to match for the transfer to understand how to map values. This is what the positions argument of multiapps is for. 

It depends what you're doing. You can transfer a single variable value, an entire field, etc. 
You just need to pick the right transfer.

**Reply by GiudGiud** [reply]
Please see this tutorial
https://mooseframework.inl.gov/getting_started/examples_and_tutorials/tutorial02_multiapps/presentation/index.html#/
